src/stock_str.rs

```diff
@@ -419,6 +419,11 @@ pub enum StockMessage {
 
     #[strum(props(fallback = "Member %1$s added."))]
     MsgAddMember = 173,
+
+    #[strum(props(
+        fallback = "⚠️ Your email provider %1$s requires end-to-end encryption which is not setup yet. Tap to learn more."
```

Comment:
even though i suggested "Tap to learn more" in _for the UI string_, i would make _the fallback core string_ more general (we do not know what UI is doing, bots etc)

```suggestion
        fallback = "⚠️ Your email provider %1$s requires end-to-end encryption which is not setup yet."
```